app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    @staticmethod
+    def __typecheck__(other: any) -> None:
```

Comment:
please, don't use `any` type annotation. I recommend you use `Union[int, float, Distance]` or `int | float | Distance` (Idk your python version)

Reply:
and fix it in every method!